workflows/smallvariants.nf

```diff
@@ -444,6 +446,26 @@ workflow SMALLVARIANTS {
             cram: [meta, cram, crai, bed]
             bam: [meta, bam, bai, bed]
         }
+    
+        //
+        // Check for MSI 
+        //
+    def ch_fasta_ready_broadcast = ch_caller_inputs.cram
+        .map { meta, _cram, _crai, _bed -> tuple(meta, file(params.fasta)) }
+
+    def ch_fai_ready_broadcast = ch_caller_inputs.cram
+        .map { meta, _cram, _crai, _bed -> tuple(meta, file(params.fai)) }
+    def ch_msi_baseline = ch_caller_inputs.cram.map { meta, _cram, _crai, _bed -> tuple(meta, file(msi_baseline)) }
+    MSISENSORPRO_PRO(
+        ch_caller_inputs.cram.map { meta, cram, crai, _bed ->
+            tuple(meta, cram, crai) },
```

Comment:
You can use `CRAM_PREPARE_SAMTOOLS_BEDTOOLS.out.ready_crams` as input to this process. What you are currently giving to as input is the same CRAM file twice (in the test config), which is not what we want here. `ch_caller_inputs` contains a multiple of entries each containing a part of the expected regions to use for variant calling. This is done to parallelize the variant calling